sqsworkers/crew.py

```diff
@@ -186,6 +195,76 @@ def poll_queue(self):
                         # continue with the next message and do not delete
                         pass
 
+    def _bulk_poll_queue(self):
+        def delete_from_sqs(entries):
+            try:
+                # delete_command = { 'Entries': entries }
+                delete_statuses = self.queue.delete_messages(Entries=entries)
+                num_deleted = len(delete_statuses['Successful']) if 'Successful' in delete_statuses else 0
+                self.crew.statsd.increment('process.record.success', num_deleted, tags=[])
+                num_not_deleted = len(delete_statuses['Failed']) if 'Failed' in delete_statuses else 0
+                if num_not_deleted > 0:
+                    self.logger.error('Delete unsuccessful for {} out of {} entries'
+                                      .format(num_not_deleted, len(entries)))
+                    for status in delete_statuses['Failed']:
+                        self.crew.statsd.increment('sqs.delete.failure', 1, tags=[])
+                        self.logger.error('delete fail for {}'.format(status))
+            except Exception as e:
+                import traceback
```

Comment:
Should we move this import to the top? And do we want to be printing exceptions? Should we maybe log.exception() so we get all the info.

Reply:
Cool. This traceback helped me resolve an issue but probably not required here anymore